query-connector/src/app/database-service.ts

```diff
@@ -267,8 +276,11 @@ export async function insertValueSet(vs: ValueSet) {
  * @returns The SQL statement for insertion
  */
 function generateValueSetSqlPromise(vs: ValueSet) {
-  const valueSetOid = vs.valueSetId;
+  const valueSetOid = vs.valueSetExternalId;
 
+  // TODO: based on how non-VSAC valuests are shaped in the future, we may need
+  // to update the ID scheme to have something more generically defined that
+  // don't rely on potentially null external ID values.
   const valueSetUniqueId = `${valueSetOid}_${vs.valueSetVersion}`;
   const insertValueSetSql =
     "INSERT INTO valuesets VALUES($1,$2,$3,$4,$5,$6) RETURNING id;";
```

Comment:
I think I missed this before, but we don't expect users to provide an `ersdConceptType`, but they should provide a `DibbsConceptType` so we should be inserting a `DibbsConceptType`

Reply:
ahh good catch! changed